kiara_rt_configure

```diff
@@ -0,0 +1 @@
+./configure --with-hydro=sphenix --disable-hand-vec --with-kernel=quartic-spline --disable-mpi --with-hydro-dimension=3 --with-rt=GEAR_3 --with-rt-riemann-solver=GLF --with-grackle=/home/zxiang/codes/grackle/local/ --with-hdf5=/home/zxiang/local/hdf5-1.12.2/hdf5/bin/h5cc --with-gearrt-grackle-mode=2 --with-subgrid=KIARA
```

Comment:
I think I just leave it here for reminding me how to compile. And I don't think I will merge branch kiara_rt into the main repository since it is used for running radiative transfer simulations with Romeel's kiara subgrid physics